test_conformance/vulkan/procs.h

```diff
@@ -0,0 +1,23 @@
+/******************************************************************
+//
+//  OpenCL Conformance Tests
+// 
+//  Copyright:	(c) 2008-2011 by Apple Inc. All Rights Reserved.
+//
+******************************************************************/
+
+#include "../../test_common/harness/mt19937.h"
```

Comment:
`harness/mt19937.h` ought to work

Reply:
Will consider this as part of the same PR while fixing build and CI issues.

Reply:
Fixed.